### DISCUSSION TITLE
Video downloads metadata - Get year metadata value from IMVDb?

### DISCUSSION THREAD

**Discussion by Yosh82** [RandomNinjaAtk/docker-lidarr-extended#235] [Ideas]
Hi,

The meta data for music video downloads are wrong for most videos pre 2010 in the nfo files. I'm unsure how it grabs the metadata, I thought it was via IMVDb, but when checking the release dates on IMVDb, they are correct on there. Is it possible to get the script to pull the "Year" value from IMVDb using the api?

https://imvdb.com/developers/api

I do see it use IMVDb in the logs, so I thought that where it was getting it's metadata from already, but the dates don't match, so it may just be connecting to it to find related videos.

Thanks,

**Comment by RandomNinjaAtk** [comment]
From reviewing the code, it appears that it uses the video upload date to get the year. The script actually grabs the imvdb year but doesn't use it at this time. I can look to make that change in the near future.

**Comment by RandomNinjaAtk** [comment]
This should fix it https://github.com/RandomNinjaAtk/docker-lidarr-extended/commit/693f13be06cb7503815f92a3b8e7c7a42b910627, but it is untested... 

You'll have to wait for the image to build with the latest version and give it a try...

**Reply by Yosh82** [reply]
I do like your definition of "near future" :) 

Thank you very much, I'll give it a test now and report back.

**Reply by Yosh82** [reply]
That looks good so far. What's the best method to replace the old nfo's? Will it just recreate them if I delete them, or will I need to delete the logs folder?

**Reply by RandomNinjaAtk** [reply]
This is just a script with no real management capabilities. If you care that much about it, you would probably need to  purge the video library data and start fresh. Not ideal, but the only way to update it...

**Reply by Yosh82** [reply]
No problem, I'll figure it out, thanks.
